### ISSUE TITLE
JWT

### ISSUE THREAD

**Issue by timowevel1** [switnet-ltd/quick-jibri-installer#87]
Hey, I got issues with JWT:

 /usr/share/lua/5.3/prosody/util/startup.lua:202: module 'basexx' not found:Failed loading module basexx in LuaRocks

luarocks install basexx doesnt fix it, do you have any idea?

**Comment by Ark74** [comment]
Hello, not really.
If you have new info on this, it will be useful to check it out.

**Comment by timowevel1** [comment]
Not really, I didnt get JWT working using your script, it should be related to any Lua stuff the script installs, maybe its outdated or packages missing, I am not sure
________________________________

**Comment by enoch85** [comment]
@Ark74 Wow, just noticed this and have been working with Jitsi for the past 6 months now. Haven't tested, but maybe could help a bit with the implementation, if this still is maintained?

**Comment by Ark74** [comment]
Sure, go ahead.
I try to keep up with the most used features, JWT haven't got much attention.
